tests/test_aws_instance.py

```diff
@@ -210,7 +210,15 @@ def test_is_aws_instance(self):
 
     def test_is_not_aws_instance(self):
         httpretty.disable()
-        assert not aws_instance.is_aws_instance()
+        real_curl = lib.scylla_cloud.curl
```

Comment:
The CI for this patch failed and AFAICS it cause bu this `lib.scylla_cloud.curl` that doesn't exist since you remove the import on the other commit

Reply:
yeah, I've reorder the commits, that's my bad.

Reply:
fixed it